### ISSUE TITLE
[BUG] Retroactive Roll Editing Doesn't trigger Dice so Nice roll

### ISSUE THREAD

**Issue by Cempres** [MangoFVTT/fvtt-ready-set-roll-5e#60]
Just as the title states. if you click on + or - for adv or disadv on an existing roll , it will not activate a dice roll from Dice so Nice

**Versions**
 - Module: [1.3.4]
 - dnd5e: [2.0.2]
 - Foundry: [10.285]

**Other Modules**
Dice so Nice

**Comment by MangoFVTT** [comment]
This is unfortunately unavoidable with the current system. Dice So Nice intercepts message renders to chat and rolls dice from that, and retroactive roll editing doesn't re-output to chat, it simply edits an existing message. Dice So Nice therefore has no indication that new dice has been rolled, and telling it otherwise would require manual interfacing with Dice So Nice which would make this module have some measure of dependency on it (which I'm not particularly keen to do).

tl;dr Currently not possible, would require more work than it's worth to implement. Potentially something to investigate at a later date.

**Comment by MangoFVTT** [comment]
Implemented a (somewhat hacky) solution for this that should make it work. It is implemented as of Release 1.3.6
